octane/evmengine/types/customproto.go

```diff
@@ -0,0 +1,120 @@
+package types
+
+import (
+	"encoding/json"
+
+	"github.com/omni-network/omni/lib/errors"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// customProtobufType defines the interface custom gogo proto types must implement
```

Comment:
```suggestion
// customProtobufType defines the interface custom go proto types must implement
```

Reply:
I think the original is correct, because this is [gogo protobuf](https://github.com/gogo/protobuf). This is a more Go opinionated implementation of protobuf, that cosmos sdk very heavily relies on.

Reply:
indeed, it is called gogo